.github/workflows/main.yaml

```
@@ -21,7 +21,31 @@ jobs:
           java-version: '11'
           distribution: 'adopt'
       - name: Building project 🛠️
-        run: ./gradlew build
+        run: ./gradlew compileJava
+  check:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+      - name: Cache gradle related files 🗄️
+        uses: actions/cache@v2.1.6
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Set up JDK 11 ⚙️
+        uses: actions/setup-java@v2.3.0
```

Bump all occurrences of setup-java to 2.3.1 and delete Dependabot pr.
```suggestion
        uses: actions/setup-java@v2.3.1
```